static/app/utils/profiling/profile/importProfile.tsx

```diff
@@ -219,11 +231,52 @@ export function importSchema(
   };
 }
 
+export function importSentryContinuousProfileChunk(
+  input: Readonly<Profiling.SentryContinousProfileChunk>,
+  traceID: string,
+  options: ImportOptions
+): ProfileGroup {
+  const frameIndex = createContinuousProfileFrameIndex(
+    input.profile.frames,
+    input.platform
+  );
+
+  return {
+    traceID,
+    name: '',
+    transactionID: null,
+    activeProfileIndex: 0,
+    profiles: [importSingleProfile(input.profile, frameIndex, options)],
+    measurements: {},
+    metadata: {
+      platform: input.platform,
+    },
+  };
+}
+
 function importSingleProfile(
-  profile: Profiling.EventedProfile | Profiling.SampledProfile | JSSelfProfiling.Trace,
-  frameIndex: ReturnType<typeof createFrameIndex>,
+  profile:
+    | Profiling.ContinuousProfile
+    | Profiling.EventedProfile
+    | Profiling.SampledProfile
+    | JSSelfProfiling.Trace,
+  frameIndex:
+    | ReturnType<typeof createFrameIndex>
+    | ReturnType<typeof createContinuousProfileFrameIndex>
+    | ReturnType<typeof createSentrySampleProfileFrameIndex>,
   {span, type, frameFilter, profileIds}: ImportOptions
 ): Profile {
+  if (isSentryContinuousProfile(profile)) {
```

Comment:
This is where things might break because the `Profiling.SampledProfile` I think also evaluates to true here.

Reply:
Yup, agree. Lets force it down this path explicitly, this is too brittle otherwise